_layouts/egazette-search.html

```diff
@@ -0,0 +1,85 @@
+---
+layout: skeleton
+---
+
+<section class="bp-section is-small search-section bp-section-pagetitle">
+  <div class="bp-container">
+    <div class="row">
+      <div class="col is-8 is-offset-2">
+        <p class="has-text-white">{%- include header-breadcrumb.html -%}</p>
+      </div>
+    </div>
+  </div>
+  <div class="bp-container">
+    <div class="row">
+      <div id="search-title" class="col is-8 is-offset-2">
+        <h1 class="has-text-white"><b>{{- page.title -}}</b></h1>
+      </div>
+    </div>
+  </div>
+  <div class="bp-container">
+    <div>
+      <div class="row">
+        <div
+          id="search-container"
+          class="col is-8 right-padding-zero is-offset-2"
+        >
+          <div
+            class="field has-addons control has-icons-left is-expanded input is-fullwidth is-large"
+            id="searchbox"
+          >
+            <!-- <div class="control has-icons-left is-expanded">
+              <input
+                class="input is-fullwidth is-large"
+                type="text"
+                placeholder="What are you looking for?"
+                autocomplete="off"
+                autocorrect="off"
+                autocapitalize="none"
+                spellcheck="false"
+              />
+              <span class="is-large is-left">
+                <i class="sgds-icon sgds-icon-search is-size-4 search-bar"></i>
+              </span> --><span class="is-large is-left">
+                <i class="sgds-icon sgds-icon-search is-size-4 search-bar"></i>
+              </span>
+            </div>
+            <div class="control">
+              <!-- <button type="submit" class="bp-button is-secondary is-medium has-text-white search-button">SEARCH</button> -->
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="bp-section">
+  <div class="bp-container search-content">
+    <div class="row search-hits-options";">
+      <div id="hits-per-page"></div>
+    </div>
+    <div class="row">
+      <!-- TODO -->
+      <div id="loading-spinner">
+			</div>
+      <div class="col is-8 is-offset-2 search-results-display">
+        <div>
+          <div class="search-summary">
+            <h4 class="search-results-count" id="search-results-count"></h4>
+            <small id="search-page-indicator"></small>
+          </div>
+          <br />
+          <h5 class="search-results" id="hits"></h5>
+        </div>
+        
+      </div>
+    </div>
+  </div>
+</section>
+
+<div id="paginator-pages">
+  <div id="pagination"></div>
```

Comment:
nit: the pagination styling is quite different from what we have on other pages, would be good to standardise them.

Reply:
Took a quick look at it - it seems like there are some styles which we specify here, but opting to delay this in favour of a quicker release - will take a look again in future if we have more capacity